app/experimenter/experiments/constants/nimbus.py

```diff
@@ -213,6 +213,15 @@ class NimbusTargetingConfig:
     application_choice_names=(Application.DESKTOP.name,),
 )
 
+TARGETING_FX95_DESKTOP_USERS = NimbusTargetingConfig(
+    name="Desktop Users on Fx95",
+    slug="fx95_desktop_users",
+    description=("Firefox 95 Desktop users"),
+    targeting=("(version|versionCompare('95.!') >= 0) && (version|versionCompare('96.!') < 0)"),
```

Comment:
Pretty sure operator precedence will check the comparisons before the AND, but this is fine.

FYI (i.e., don't update patch) If you really wanted to "optimize" 😝  and maximize the mozilla version comparison quirks.. `version|versionCompare('95+') < 0` works for the second part and `version|versionCompare('95pre') >= 0` for the first………

Reply:
Yeah I would've thought so too but it didn't work in my local testing without them

Reply:
Also interesting good to know 👍